modules/data.atmosphere/R/solar_angle.R

```diff
@@ -11,18 +55,18 @@
 #' @return `numeric(1)` of cosine of solar zenith angle
 #' @export
```

Comment:
*usually* `@examples` goes at the end of the roxygen2 comments block.  There's no hard rule about that, but since examples appear last in rendered documentation, it makes sense to put them last here as well.